docs/AfterAppointment.puml

```diff
@@ -5,10 +5,10 @@ object "<u>:Main</u>" as Main
 object "<u> :CommandHandler</u>" as CommandHandler
 object "<u> in:Scanner</u>" as Scanner
 object "<u> appointment:Appointment </u>" as Appointment {
-name = John Doe
-nric = S123A
-date = 18-11-2024
-time = 18:00
+name = "John Doe"
+nric = "S123A"
+date = "18-11-2024"
+time = "18:00"
```

Comment:
Perhaps don't need quotation marks for date and time since it's not a string, it's a LocalDate and LocalTime respectively